networking_calico/etcdutils.py

```diff
@@ -389,7 +389,7 @@ def stop(self):
         self._stopped = True
 
 
-def intern_dict(d, fields_to_intern=None):
+def intern_dict(d):
     """intern_dict
 
     Return a copy of the input dict where all its string/unicode keys
```

Comment:
Comment out of date, looks like interning values is no longer supported

Reply:
Discussed in another channel: the point here is that it is no longer optional that some field values are interned.  I've updated the comment accordingly.